client/components/calendar/calendar.jsx

```diff
@@ -91,23 +91,24 @@ class Calendar extends Component {
     })
   }
 
-  render(){
-    let weeksArray = ['sun', 'mon', 'tue', 'wed', 'thurs', 'fri', 'sat'];
+  render() {
+    let weeksArray = ['Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa'];
 
     return (
-      <div style={Styles.main}>
-        <p style={Styles.monthText}>{this.state.month}</p>
-        <p style={Styles.yearText}>{this.state.year}</p>
-        <table>
+      <div className="main">
+        <div className="row">
+          <button className="button-left" onClick={this.dateLeft}>←</button>
+          <strong>{this.state.month} {this.state.year}</strong>
+          <button className="button-right" onClick={this.dateRight}>→</button>
+        </div>
+        <table className="table">
           <tbody>
             <tr>
-              {weeksArray.map((day, i) => <td key={i} style={Styles.calendarText}>{day}</td>)}
+              {weeksArray.map((day, i) => <td key={i} className="calendar-text">{day}</td>)}
             </tr>
-            {this.state.weeks.map((week, i) => <Week key={i} dates={week} incrementMonth={this.state.incrementMonth}/>)}
+            {this.state.weeks.map((week, i) => <Week unavailableDates={this.props.unavailableDates} checkIn={this.props.checkIn} checkOut={this.props.checkOut} calType={this.props.calType} getDate={this.props.getDate} key={i} dates={week} incrementMonth={this.state.incrementMonth} />)}
```

Comment:
Avoid cramming so much code into one line. A linter can help you identify these instances. https://github.com/airbnb/javascript/tree/master/react#alignment